README.md

```diff
@@ -2,46 +2,67 @@
 
 This Node.js SDK provides integration with Yaya Wallet API, allowing developers to interact with the Yaya Wallet platform programmatically.
 
-## Installation
+## Getting Started with the Yaya Wallet SDK for Node.js
 
-You can install the Yaya Wallet Node SDK via npm:
+This guide will walk you through setting up and using the Yaya Wallet SDK in your Node.js application to interact with the Yaya Wallet API.
+
+**1. Installation**
+
+Before you begin, ensure you have Node.js and npm installed on your system. You can install the SDK via npm using the following command:
 
 ```bash
 npm install @yayawallet/node-sdk
 ```
 
-## Usage
+**2. Environment Setup**
+
+**Secure Credentials with a .env File:**
+
+1. Create a file named .env in the root directory of your project. This file will store your API credentials securely, separate from your code.
 
-To use the SDK in your Node.js application, require it as follows:
+2. Add the following lines to the .env file, replacing the placeholders with your actual Yaya Wallet API credentials:
 
 ```bash
-const yayaSdk = require('@yayawallet/node-sdk');
+API_URL=https://yayawallet.com/api/en
```

Comment:
Does not these keys have `YAYA` prefix?

Reply:
They had not, but in the last commit, @ermiyas111 has prefixed it with YAYA. Anyways, I am gonna change it.